app/assets/stylesheets/provider/_vertical_nav.scss

```
@@ -7,12 +7,18 @@
   }
 
   // scss-lint:disable SelectorFormat
-  .pf-c-nav__item.outdated-config::before {
-    content: '\f071';
-    font-family: FontAwesome;
-    font-weight: normal;
-    position: absolute;
-    right: line-height-times(3);
-    top: line-height-times(1 / 3);
+  .pf-c-nav__item {
+    &.outdated-config > a::before {
```

If my suggestion below is rejected I think this one might improve readability:
```suggestion
    &.outdated-config:not(.pf-m-expanded) > a::before {
```